### ISSUE TITLE
Priority 2 Accessibility issue: Newsletter form email validation status

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6740]
**What**

When attempting to subscribe to the newsletter and submitting the ‘email address’ field with no input or invalid input, an error message appears; however, this message is not announced to screen reader users. As such, screen reader users may not be aware of the error when attempting to submit the form.

Solution:
Ensure that all users are made aware of error messages when they appear. This can be achieved by using an aria-live region, utilising a role of status on the error message container. Please note that the container must exist within the DOM prior to the message appearing so that an update is flagged, to allow the status to announce. Additionally, it would benefit users if the error message only appeared when the field is submitted.

**Where**

https://wellcomecollection.org/
